com.unity.formats.alembic/Tests/Editor/MeshSchemaTests.cs

```diff
@@ -0,0 +1,315 @@
+        };
+
+        static readonly Dictionary<string, List<Vector3>> k_VertexNormalScopeTestData = new Dictionary<string, List<Vector3>>()
+        {
+            { "face_normal_grid", new List<Vector3>
+            {
+                new Vector3(0,0,1),
+                new Vector3(0,0,1),
+                new Vector3(0,0,1),
+                new Vector3(0,0,1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 1, 0),
+                new Vector3(0, 1, 0),
+                new Vector3(0, 1, 0),
+                new Vector3(0, 1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(-1, 0, 0),
+                new Vector3(-1, 0, 0),
+                new Vector3(-1, 0, 0),
+                new Vector3(-1, 0, 0)
+            }},
+            { "point_normal_grid", new List<Vector3>
+            {
+                new Vector3(0.57735f, -0.57735f, 0.57735f),
+                new Vector3( -0.57735f, -0.57735f, 0.57735f),
+                new Vector3(  -0.57735f, 0.57735f, 0.57735f),
+                new Vector3( 0.57735f, 0.57735f, 0.57735f),
+                new Vector3(-0.57735f, -0.57735f, -0.57735f),
+                new Vector3(0.57735f, -0.57735f, -0.57735f),
+                new Vector3(0.57735f, 0.57735f, -0.57735f),
+                new Vector3(-0.57735f, 0.57735f, -0.57735f)
+            }},
+            { "vertex_normal_grid", new List<Vector3>
+            {
+                new Vector3(0, 0, 1),
+                new Vector3(0, 0, 1),
+                new Vector3(0, 0, 1),
+                new Vector3(0, 0, 1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 0, -1),
+                new Vector3(0, 1, 0),
+                new Vector3(0, 1, 0),
+                new Vector3(0, 1, 0),
+                new Vector3(0, 1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(0, -1, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(1, 0, 0),
+                new Vector3(-1, 0, 0),
+                new Vector3(-1, 0, 0),
+                new Vector3(-1, 0, 0),
+                new Vector3(-1, 0, 0)
+            }}
+        };
+
+        [Test]
+        [TestCase("cube_face")]
+        [TestCase("cube_point")]
+        [TestCase("cube_vertex")]
+        public void VertexRgb_AreProcessedCorrectlyForScope(string scope)
```

Comment:
naming of the test functions could be clearer
```suggestion
        public void Import_InScope_VertexRgb_Colors_AreAsExpected(string scope)
```

Reply:
True ! changed here [f30b011](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/569/commits/f30b011d14d8c41cf329c4d40526a62db738873e)